csharp/device/Microsoft.Azure.Devices.Client/DeviceClient.cs

```diff
@@ -139,7 +140,7 @@ public static DeviceClient CreateFromConnectionString(string connectionString, T
 #if WINDOWS_UWP
                 throw new NotImplementedException();
 #else
-                return new DeviceClient(new AmqpDeviceClient(iotHubConnectionString));
+                //return new DeviceClient(new AmqpDeviceClient(iotHubConnectionString));
```

Comment:
This one is easy: the AMQP is still on the works. See the pr message, only the HTTP part is working.